Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -38,9 +40,13 @@ public partial class MasterDetailsView : ItemsControl
         private const string NoSelectionNarrowState = "NoSelectionNarrow";
         private const string NoSelectionWideState = "NoSelectionWide";
 
-        private AppViewBackButtonVisibility _previousBackButtonVisibility;
+        private AppViewBackButtonVisibility? _previousSystemBackButtonVisibility;
+        private bool _previousNavigationViewBackEnabled;
+        private NavigationViewBackButtonVisible _previousNavigationViewBackVisibilty;
```

Comment:
This property is not available on 17134 - this would cause the control to crash

Reply:
Is there a property available that the new NavView uses to go back that far??

Reply:
Aren't all the properties for the new WinUI NavView part of the WinUI nuget?

Reply:
The WinUI NavView uses the same property/type from the framework control